src/components/sections/ProductGallery/ProductGallery.tsx

```diff
@@ -38,7 +40,25 @@ function ProductGallery({ title }: Props) {
     setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
   }, [screenWidth])
 
-  if (!data) {
+  const orderedFacets: FacetedFilter_FacetsFragment[] = useMemo(() => {
+    if (data) {
+      const orderFacets = data.search.facets.map((facet) => {
+        if (facet.type === 'BOOLEAN') {
+          facet.values = facet.values.sort((a, b) =>
+            a.label.localeCompare(b.label)
+          )
+        }
+
+        return facet
+      })
+
+      facets.current = orderFacets
+    }
+
+    return facets.current
+  }, [data])
```

Comment:
We should avoid this kind of processing in the front-end. We don't need to tackle this now but I think it's good to create a Jira ticket to try to send this behavior to the server.